Multi binding sample

lachlanwgordon opened the pull request:
**DO NOT MERGE** This PR relies on changes in #218, make sure it is merged in before this one. 

### Description of Change ###

Adds a sample for Multiconverters. It is closely based on the sample from the comments on the original PR #137 .

The sample demonstrates using the `MuliConverter` in conjunction with the `NotEqualConverter` and the `TextCaseConverter` to check that an entered string is not the same as a given string, converting the entered string to upper case.

![2020-08-20 22 17 36](https://user-images.githubusercontent.com/29908924/90769127-15650280-e333-11ea-991a-691ebd8355b4.gif)

### Bugs Fixed ###

- Fixes #168 
- Adds sample for #137 
- Tests already added in #156

### API Changes ###

None

Added: 
- MultiConverterPage
- MultiConverterViewModel
- MultiConverterDescription in resx
- MultiConverterShortDescription in resx

Changed:

 - `Items` in ConverterGalleryViewModel to include new sample

### Behavioral Changes ###

None

### PR Checklist ###

- [ ] Has tests (if omitted, state reason in description)
- [ ] Has samples (if omitted, state reason in description)
- [ ] Rebased on top of main at time of PR
- [ ] Changes adhere to coding standard
- [ ] Updated documentation

jfversluis commented:
This is ready to go now, right?

One thing that is hurting my eyes... Would you be able to add some padding on the left and right so it will look a tiny bit prettier? 😬

lachlanwgordon commented:
It was "ready" (except for a merge conflict that I've fixed now).

I've added some padding and spacing to make it a little prettier and update.

The Gif above has been updated.
